Correct key `cluster.kubernetes.error.configNotFoundError` in cluster_kubernetes.en.toml

## Description

**Version**: <!-- Please add the CLI version number here --> 

<!-- Please provide as much detail about the bug as possible.-->

### Steps to reproduce
1. Do `rhoas cluster status`

### Fix
change the key name here - https://github.com/redhat-developer/app-services-cli/blob/main/pkg/localize/locales/en/cluster_kubernetes.en.toml#L16

### Logs

```bash
❯ ./rhoas cluster status
panic: message "cluster.kubernetes.error.configNotFoundError" not found in language "en"

goroutine 1 [running]:
github.com/nicksnyder/go-i18n/v2/i18n.(*Localizer).MustLocalize(0x1400054d6a0, 0x1400020da78, 0x105ea88a0, 0x1400059a240)
        /Users/ankithans/go/pkg/mod/github.com/nicksnyder/go-i18n/v2@v2.1.2/i18n/localizer.go:211 +0x70
github.com/redhat-developer/app-services-cli/pkg/localize/goi18n.(*Goi18n).MustLocalize(0x140004f99a0, 0x1059fd1e6, 0x2c, 0x0, 0x0, 0x0, 0x28e, 0x14000650f00)
        /Users/ankithans/Documents/Projects/go/app-services-cli/pkg/localize/goi18n/go_i18n.go:85 +0x124
github.com/redhat-developer/app-services-cli/pkg/cluster.NewKubernetesClusterConnection(0x1060746a0, 0x140002dd440, 0x106082bd0, 0x106c91408, 0x106091970, 0x14000573aa0, 0x140000fd180, 0x1d, 0x1400059f900, 0x106057310, ...)
        /Users/ankithans/Documents/Projects/go/app-services-cli/pkg/cluster/kubernetes_cluster.go:75 +0x88
github.com/redhat-developer/app-services-cli/pkg/cmd/cluster/status.runStatus(0x1400059fa40, 0x1400020dc98, 0x0)
        /Users/ankithans/Documents/Projects/go/app-services-cli/pkg/cmd/cluster/status/status.go:69 +0xf0
github.com/redhat-developer/app-services-cli/pkg/cmd/cluster/status.NewStatusCommand.func1(0x14000355680, 0x106c91408, 0x0, 0x0, 0x0, 0x0)
        /Users/ankithans/Documents/Projects/go/app-services-cli/pkg/cmd/cluster/status/status.go:49 +0x28
github.com/spf13/cobra.(*Command).execute(0x14000355680, 0x106c91408, 0x0, 0x0, 0x14000355680, 0x106c91408)
        /Users/ankithans/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x320
github.com/spf13/cobra.(*Command).ExecuteC(0x140002b0500, 0x105bf3458, 0x3, 0x140002b0500)
        /Users/ankithans/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x264
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/ankithans/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
main.main()
        /Users/ankithans/Documents/Projects/go/app-services-cli/cmd/rhoas/main.go:54 +0x2e8
```

--- a/pkg/localize/locales/en/cluster_kubernetes.en.toml
+++ b/pkg/localize/locales/en/cluster_kubernetes.en.toml
@@ -13,7 +13,7 @@ Kubernetes Namespace:   {{.Namespace}}
 Service Account Secret: {{.ServiceAccountSecretName}}
 '''
 
-[cluster.kubernetes.error.missingConfigError]
+[cluster.kubernetes.error.configNotFoundError]
 one = 'missing kubeconfig file'
 
 [cluster.kubernetes.error.loadConfigError]

